src/views/cards/cards.scss

```diff
@@ -0,0 +1,39 @@
+@import "../../colors";
+@import "../../typography";
+
+.intro {
```

Comment:
I think we already have an element with this class name — could you wrap all of the styles for the component in a class so there can't be any interaction? I know practically there won't be, but just for consistency.  E.g., add a `cards` class to the `div.inner`, and then wrap all these styles in `.cards`?